charms/worker/k8s/scripts/update_alert_rules.py

```diff
@@ -52,11 +52,11 @@ def download_and_process_rule_files(temp_dir: Path):
         source_url = f"{SOURCE}/{file}"
         temp_file = temp_dir / file
         try:
-            logging.info(f"Downloading {source_url}")
+            logging.info("Downloading %s", source_url)
```

Comment:
proper way to use a logger... formatting is done in the logger only if that log level is active